src/utils/IsSame.ts

```diff
@@ -1,4 +1,5 @@
-import { isEqual, omit } from 'lodash'
+import { isEqual } from 'lodash'
+import omit from 'lodash.omit'
```

Comment:
Same as Create UI, where the build would tell me "Omit not found. Did you mean ...". This works.